src/components/Forum/ItemDetail.js

```diff
@@ -17,44 +17,36 @@ export function ItemDetails(props) {
   const [commentText, setCommentText] = React.useState("");
```

Comment:
Why are you importing useState as indepent from React and then use React.useState instead of just useState?